sky/provision/kubernetes/network_utils.py

```diff
@@ -75,13 +75,16 @@ def fill_loadbalancer_template(namespace: str, service_name: str,
 
     with open(template_path, 'r', encoding='utf-8') as fin:
         template = fin.read()
+    annotations = skypilot_config.get_nested(
```

Comment:
We should also do this for ingress (in `fill_ingress_template()`). We can add it in this PR, or put a TODO and file an issue to keep track of it :)